manifests/a/akon47/ScreenRecorder/1.1.2/akon47.ScreenRecorder.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: akon47.ScreenRecorder
+PackageVersion: 1.1.2
+PackageLocale: en-US
+Publisher: akon47

```

```suggestion
Publisher: kimhwan
```
This must match Control Panel.

---

very very thank you!